piker/config.py

```diff
@@ -17,21 +17,109 @@
 """
 Broker configuration mgmt.
 """
+import platform
+import sys
 import os
 from os.path import dirname
 import shutil
 from typing import Optional
 
 from bidict import bidict
 import toml
-import click
 
 from .log import get_logger
 
 log = get_logger('broker-config')
 
-_config_dir = click.get_app_dir('piker')
+
+# taken from ``click`` since apparently they have some
+# super weirdness with sigint and sudo..no clue
+def get_app_dir(app_name, roaming=True, force_posix=False):
```

Comment:
Ahh right, yeah this is probably confusing af..

It seems that using SIGIN cancellation was somehow super oddly being swallowed/ignored by `click` and so pulling out this one routine we needed (which probably isn't a horrible idea to maintain on our own anyway?) was the fast hack.